NitroxClient/Communication/Packets/Processors/WeldActionProcessor.cs

```diff
@@ -0,0 +1,52 @@
+﻿using NitroxClient.Communication.Abstract;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    class WeldActionProcessor : ClientPacketProcessor<WeldAction>
+    {
+        private IMultiplayerSession multiplayerSession;
+        private SimulationOwnership simulationOwnership;
+
+        public WeldActionProcessor(IMultiplayerSession multiplayerSession, SimulationOwnership simulationOwnership)
+        {
+            this.multiplayerSession = multiplayerSession;
+            this.simulationOwnership = simulationOwnership;
+        }
+
+        public override void Process(WeldAction packet)
+        {
+            Optional<GameObject> opGameObject = NitroxEntity.GetObjectFrom(packet.Id);
```

Comment:
Alternatively, I think there is a RequireObjectfFrom which will internally throw.  That will stop only this packet processor and is safe to do.